spec.html

```diff
@@ -28934,7 +28960,8 @@ <h1>
         </dl>
         <p>
           _parameterStrings_ represents the strings that, when using one of the function constructors, will be concatenated together to build the parameters list. _bodyString_ represents the function body or the string passed to an `eval` call.
-          _direct_ signifies whether the evaluation is a direct eval.
+          _codeString_ represents the full code string. _compilationType_ is an enum that indicates what type of compilation this is.
```

Comment:
> \_compilationType\_ is an enum that indicates what type of compilation this is.

This sentence is nearly tautological. Either explain what each of the 3 possible values means or omit the sentence.